jackal_move/goal/src/objective.txt

```diff
@@ -0,0 +1,4 @@
+2 -2 0
+2 2 1.5708
+-2 2 3.1416
+-2 -2 4.7124
```

Comment:
Are these the waypoints? What axes do they correspond to? Can you put a comment above briefly explaining